examples/olm/install-resources.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: v1
 kind: Namespace
 metadata:
     name: security-profiles-operator
-    openshift.io/cluster-monitoring: "true"
```

Comment:
Why not put the cluster-monitoring into labels? But if you just want to fix the tests in the fastest way possible, then this is fine.